### PR TITLE
Add wishlists menu item for logged in users

### PR DESCRIPTION
**Pull request by gurch101**

### CODE REVIEW

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/base.jinja [hunk 1]
```diff
@@ -14,6 +14,8 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
     {# Include Styles #}
     <link rel="stylesheet" href="{{ STATIC_URL }}classic_gray/css/style.css">
+    {# Include all Extra Styles #}
+    {% block extrastyles %}{% endblock %}
```
Comment (shawnadelic) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/base.jinja]: How is this being used?

There is also the add_resource context function in xtheme/resources.py which can insert JS or CSS into the head if you're just trying to add styles to a plugin. Not sure if that fits your needs.

Comment (gurch101) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/base.jinja]: thanks @shawnadelic! add_resource was exactly what I needed.

**Comment by tulimaki** [comment]
ok to test

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/_navigation.jinja [hunk 2]
```diff
@@ -43,6 +43,14 @@
                             </a>
                         </li>
                         {% endif %}
+                        {% if shoop.urls.has_url("shoop:personal_wishlists") %}
+                        <li>
+                            <a href="{{ url("shoop:personal_wishlists") }}">
+                                <i class="fa fa-star fa-fw"></i>
+                                {% trans %}My wishlists{% endtrans %}
```
Comment (tulimaki) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/_navigation.jinja]: Missing translations.

**Comment by tulimaki** [comment]
Also add changelog item for this change

File: doc/changelog.rst [hunk 3]
```diff
@@ -109,6 +109,7 @@ Addons
 Front
 ~~~~~
 
+- Add Shoop Wishlist addon support for logged in users
```
Comment (tulimaki) [inline doc/changelog.rst]: This  log entry should be under classic gray theme

**Comment by tulimaki** [comment]
Also squash these commits...

**Pull request closed without merging**